cleverhans/model.py

```diff
@@ -11,9 +11,18 @@ class Model(object):
     model when a specific package does not directly expose them.
     """
     __metaclass__ = ABCMeta
+    O_LOGITS, O_PROBS, O_FEATURES = 'logits probs features'.split()
 
-    def __init__(self):
-        pass
+    def __init__(self, scope, nb_classes, hparams):
```

Comment:
This breaks the prior API but does not seem to be necessary for the existing code (unless I missed something). If that's true, could you set default values to be None and add a deprecation warning?

Reply:
Yes it's not necessary, I'll put some defaults.